frontend/employer/src/hooks/useCompanyQuery.ts

```diff
@@ -0,0 +1,14 @@
+import useBackendAPI from 'employer/hooks/useBackendAPI';
+import Company from 'employer/types/company';
+import { useQuery, UseQueryResult } from 'react-query';
+
+export const COMPANY_URL = '/v1/company/';
```

Comment:
maybe we can have a separate constants file for urls, as i guess we will have many endpoints, so easy to manage

Reply:
Usage will be always with hooks, `useXXXQuery` so the endpoint can be hide inside then. On the other hand I still needed to export it to the test, so perhaps It could be a good idea to separate them to a contant/enum